install.sh

```diff
@@ -829,11 +829,27 @@ do_unmount_and_remove() {
     set -x
 }
 
+clean_mounted_directory() {
+    for path in "$1"/*; do
+        if [ -d "$path" ]; then
+            if grep -q "$path" /proc/mounts; then
```

Comment:
nit - should we match on the specific path, or are we intentionally matching anything that **contains** this path?
```suggestion
            if grep -q " $path " /proc/mounts; then
```

Reply:
intentionally matching anything that has the path inside, but with the whitespace before.. because since it is a recursion I don't want to delete folders that have mounted folders inside.